quarkus/runtime/src/main/java/org/keycloak/cli/MainCommand.java

```diff
@@ -117,7 +118,8 @@ private void beforeReaugmentationOnWindows() throws Exception {
             mixinStandardHelpOptions = true,
             optionListHeading = "%nOptions%n",
             parameterListHeading = "Available Commands%n")
-    public void startDev(@Option(names = "--verbose", description = "Print out more details when running this command.", required = false) Boolean verbose) {
+    public void startDev(@Option(names = "--verbose", description = "Print out more details when running this command.", required = false) Boolean verbose,
+            @Option(names = AUTO_BUILD_OPTION, description = "Automatically detects whether the server configuration changed and a new server image must be built prior to starting the server. This options provides an alternative to manually running the '" + BUILD_COMMAND + "' prior to starting the server. Use this configuration carefully in production as might impact the startup time.", required = false) Boolean autoConfig) {
```

Comment:
Typo: "This options provides an alternative" -> "This option provides an alternative"

Typo: "Use this configuration carefully in production as might impact the startup time" -> "Use this configuration carefully in production as it might impact the startup time"